Kickstarter-iOS/Views/Storyboards/SettingsAccountWarningCell.xib

```diff
@@ -20,13 +20,17 @@
                     <rect key="frame" x="10" y="26" width="355" height="635"/>
                     <subviews>
                         <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="koj-7J-qDc">
-                            <rect key="frame" x="0.0" y="307.5" width="288" height="20.5"/>
+                            <rect key="frame" x="0.0" y="307.5" width="286" height="20.5"/>
                             <fontDescription key="fontDescription" type="system" pointSize="17"/>
                             <nil key="textColor"/>
                             <nil key="highlightedColor"/>
                         </label>
                         <imageView clearsContextBeforeDrawing="NO" userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="253" verticalHuggingPriority="253" horizontalCompressionResistancePriority="752" verticalCompressionResistancePriority="752" image="icon--alert" translatesAutoresizingMaskIntoConstraints="NO" id="W3R-uz-66d">
-                            <rect key="frame" x="306" y="306.5" width="22" height="22"/>
+                            <rect key="frame" x="304" y="305.5" width="24" height="24"/>
```

Comment:
Since we're now using smaller image we have to tell the `image view` to explicitly size itself. Since it's using `.fill` `contentMode` it will adjust for the difference and the UI will be almost identical (I say almost because the image offset changes by 1 pixel - which should be fine but requires snapshot update).